scratch/PCA/extract_data.ipynb

```diff
@@ -18,40 +18,81 @@
     "\"\"\"\n",
     "\n",
     "from __future__ import division\n",
```

Comment:
Not the best place for this note, but I can't comment on the file as a whole. Parsing out ipython notebooks isn't really doable in a pull request -- the JSON formatting makes it to difficult to read. We can go through this as a demo. Do you have 30 minutes later today, say at 11:00? I'll be in my office, if I've managed to get Yasuhito going on the XCT, otherwise I'll be in W230.